new-starter-guide/new-starter-to-origami.md

```diff
@@ -11,16 +11,18 @@ The checklist assumes that you have done the following:
 
 ### Admin privileges on your machine
 
-<abbr title="IT Services Desk">ITSD</abbr> should have enabled admin privileges when they have set up your machine but you should [request admin access to your FT laptop via Salesforce](https://financialtimes.my.salesforce.com/home/home.jsp) to ensure it does not get revoked.
+<abbr title="IT Services Desk">ITSD</abbr> should have done admin privileges for you and your machine when they did the setup. If not, then you should [request admin access to your FT laptop via Salesforce](https://financialtimes.my.salesforce.com/home/home.jsp) to ensure it does not get revoked.
```

Comment:
Yes, I think that would make it much clearer (Hemingway, go and sit in the naughty corner and don't come back till you apologise).